### PR TITLE
feat(docs): add NFT cookbook

### LINKED ISSUE #322: Working with NFTs in Tact
Description in https://docs.tact-lang.org/cookbook/nfts is missing. It should contain recipes on how to work with NFT contracts in Tact. (Note this is not about implementing NFTs in Tact)

### LINKED ISSUE #957: Working with NFTs in Tact
Clone of https://github.com/tact-lang/tact-docs/issues/322

### PR DESCRIPTION
**Pull request by a-bahdanau**
## Issue

Closes #957 and Closes https://github.com/tact-lang/tact-docs/issues/322

## Checklist

- [x] I have updated CHANGELOG.md
- [x] I have run the linter, formatter and spellchecker
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

**Comment by Gusarich** [comment]
Didn't we already review this one in tact-docs repo? Should we review it again, what was changed?

File: docs/src/content/docs/cookbook/nfts.mdx [hunk 1]
```diff
@@ -3,8 +3,243 @@ title: Non-Fungible Tokens (NFTs)
...
+        }.toCell(),
+    });
+}
+```
+
+:::info
+
+  Didn't find your favorite example of a NFT communication? Have cool implementations in mind? [Contributions are welcome!](https://github.com/tact-lang/tact-docs/issues)
+
+::info
```
Comment (novusnota) [inline docs/src/content/docs/cookbook/nfts.mdx]:
```suggestion
:::tip[Hey there!]

  Didn't find your favorite example of a NFT communication? Have cool implementations in mind? [Contributions are welcome!](https://github.com/tact-lang/tact/issues)

:::
```

File: docs/src/content/docs/cookbook/nfts.mdx [hunk 2]
```diff
@@ -3,8 +3,243 @@ title: Non-Fungible Tokens (NFTs)
...
+```
+
+## NFT Collection methods
+
+
+:::caution
+
+  These methods are not part of any standard, and they will only work with [this specific implementation](https://github.com/ton-blockchain/token-contract/blob/main/nft/nft-collection.fc). Please keep this in mind before using them.
+
+:::caution
```
Comment (novusnota) [inline docs/src/content/docs/cookbook/nfts.mdx]:
```suggestion
:::
```

File: docs/src/content/docs/cookbook/nfts.mdx [hunk 3]
```diff
@@ -3,8 +3,243 @@ title: Non-Fungible Tokens (NFTs)
 description: "Common examples of working with Non-Fungible Tokens (NFTs) in Tact"
 ---
 
-:::danger[Not implemented]
+# Non-Fungible Tokens (NFTs)
```
Comment (novusnota) [inline docs/src/content/docs/cookbook/nfts.mdx]:
```suggestion
```

**Review by novusnota — CHANGES_REQUESTED** [review]
Yeah, content-wise this PR was already approved by @Gusarich, so nothing changed there. I just want to clean-up the formatting for the new docs and we can merge this

**Review by novusnota — APPROVED** [review]

**Pull request merged**